### PR TITLE
Adding HRP article

### PR DESCRIPTION
**Pull request by nextlevelanalytics**

### CODE REVIEW

**Comment by nextlevelanalytics** [comment]
Hi Dirk, I invoked `make` and made the HRP article, but the table made using kable to display the time is not showing. I thought it had to do with knitr:: not being there, but I've added that and ran `make` again, but kable is still not displaying. Do you know why that is?

**Comment by eddelbuettel** [comment]
See what I used for Gerber :

``` r
knitr::kable(result[,1:4, with = FALSE])
```

Can you try that? 

_Edit:_ I see you did. In that case I do not know off-hand.  Start with a simpler / smaller document?

File: _posts/2016-05-27-HRP.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
+---
+title: "Hierarchical Risk Parity Implementation in Rcpp and OpenMP"
+author: Rafael Nicolas Fermin Cota, Yi King, and Chris Chong
+license: GPL (>= 2)
+tags: openmp recursion modelling optimization
```
Comment (eddelbuettel) [inline _posts/2016-05-27-HRP.md:5]: modeling, with one l, is the tag we use.

**Comment by Ecthilisyk** [comment]
Hi Dirk, just to confirm, are you still experiencing the segfault error?

**Comment by eddelbuettel** [comment]
I have not had a chance to try this submission,

**Comment by nextlevelanalytics** [comment]
Hi Dirk,
HRP has been updated, can you try that new Rmd instead?

**Comment by eddelbuettel** [comment]
You didn't fix the 'modeling' spelling :-/

**Pull request merged**